Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -409,6 +409,12 @@ open class _ChatMessageContentView<ExtraData: ExtraDataTypes>: _View, ThemeProvi
 
         NSLayoutConstraint.activate(constraintsToActivate)
     }
+    
+    override open func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
```

Comment:
Because otherwise the view doesn't properly respond to dynamic font changes.
Also there are a couple of snapshot tests for system messages that were recorded incorrectly for small fonts because this was abscent, re-recorded them in this PR